examples/bigcommerce/README.md

```diff
@@ -131,44 +131,56 @@ BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
 
 With a home page and product template pages created it's probably a good time to explain what's going on.
 
+This explanation is loaded with Next.js terms. Here is a point of reference if you are unfamiliar:
```

Comment:
This is not a big deal, but "dynamic product route" is not a Next.js term, it's a combination of Next.js and a page specific to this example. If you just said "dynamic route" then that would be 100% Next.js... This is just a nit, but it jumped out to me 😄